### PR TITLE
update package names, fixes #3

### LINKED ISSUE #3: Does Not Build using ./cover.sh
I ran glide:

```
$ glide install
[WARN]	The name listed in the config file (github.com/30x/apidApigeeSync) does not match the current location (github.com/apid/apidCounters)
[INFO]	Lock file (glide.lock) does not exist. Performing update.
[WARN]	The name listed in the config file (github.com/30x/apidApigeeSync) does not match the current location (github.com/apid/apidCounters)
[INFO]	Downloading dependencies. Please wait...
[INFO]	--> Fetching updates for github.com/30x/apid-core
[INFO]	--> Fetching updates for github.com/onsi/gomega
[INFO]	--> Fetching updates for github.com/onsi/ginkgo
[INFO]	Resolving imports
```

(continues from there) 

Then ran `./cover.sh`:

```
$ ./cover.sh
# apidCounters
/var/folders/st/4_nh2vbs28bclhkb2d2j0chw00d89p/T/go-build870463207/apidCounters/_test/_obj_test/init.go:22:21: not enough arguments in call to "apidCounters/vendor/github.com/30x/apid-core".RegisterPlugin
	have (func("apidCounters/vendor/github.com/30x/apid-core".Services) ("apidCounters/vendor/github.com/30x/apid-core".PluginData, error))
	want ("apidCounters/vendor/github.com/30x/apid-core".PluginInitFunc, "apidCounters/vendor/github.com/30x/apid-core".PluginData)
./apidCounters_suite_test.go:20:48: cannot use factory.DefaultServicesFactory() (type "apidCounters/vendor/github.com/apid/apid-core".Services) as type "apidCounters/vendor/github.com/30x/apid-core".Services in argument to "apidCounters/vendor/github.com/30x/apid-core".Initialize:
	"apidCounters/vendor/github.com/apid/apid-core".Services does not implement "apidCounters/vendor/github.com/30x/apid-core".Services (wrong type for API method)
		have API() "apidCounters/vendor/github.com/apid/apid-core".APIService
		want API() "apidCounters/vendor/github.com/30x/apid-core".APIService
./apidCounters_suite_test.go:34:24: not enough arguments in call to "apidCounters/vendor/github.com/30x/apid-core".InitializePlugins
	have ()
	want (string)
FAIL	apidCounters [build failed]
```

* `go version go1.9.2 darwin/amd64`
* `glide version 0.13.1`
* MacOS 10.12.6

### PR DESCRIPTION
**Pull request by theganyo**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,7 +1,7 @@
 language: go
 
 go:
-  - 1.7.x
+  - 1.9.x
```
Comment (chebrolus) [inline .travis.yml]: I see 1.8.3 in most other APID components. If you depend on golang 1.9.x then should this be updated for rest of APID components?

**Comment by theganyo** [comment]
1.9 isn't required and it doesn't matter to the rest of the system as this is just example code, but I added go 1.8.3 for you for consistency and proof. :)

**Review by sramamoorthy73 — APPROVED** [review]
LGTM

**Comment by chebrolus** [comment]
Great thank you @theganyo 
Yes you are right about this being example plugin it may not be big deal other than for consistency.

**Pull request merged**